src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsProjectBuildProperties.cs

```diff
@@ -74,6 +86,7 @@ public string GetPropertyValue(string propertyName)
                     _project = _dteProject.Value;
                 }
                 var property = _project.Properties.Item(propertyName);
+                _buildPropertiesTelemetry.OnDteUsed(_projectTypeGuids);
```

Comment:
The way the code is currently written, wouldn't we hit this every time a property isn't set? 

Wouldn't that make the current data not super helpful?
Should we instead be looking for differences between the 2 results instead? In particular, when the property storage was checked, but the result from it was different from the DTE one.

Reply:
No, `DTE.Project.Properties.Item` throws an `ArgumentException` when the property isn't found (which is exactly what the VS perf team want us to "fix").